src/pages/settings/PasswordPage.js

```diff
@@ -156,7 +156,14 @@ class PasswordPage extends Component {
                         onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_SECURITY)}
                         onCloseButtonPress={() => Navigation.dismissModal(true)}
                     />
-                    <ScrollView style={styles.flex1} contentContainerStyle={styles.p5}>
+                    <ScrollView
+                        style={styles.flex1}
+                        contentContainerStyle={styles.p5}
+
+                        // to allow children of the scroll view to catch taps. when keyboard is opened.
+                        // eslint-disable-next-line react/jsx-props-no-multi-spaces
```

Comment:
We know what this prop does. Comment should reflect why this is used.

Reply:
ok, I think we can change it to 
// to allow the user to press the show password icon when keyboard is open.

Reply:
Allow the user to click show password while password input is focused.

Reply:
ok done